report_py3o_fusion_server/models/py3o_pdf_options.py

```diff
@@ -0,0 +1,316 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+import logging
+logger = logging.getLogger(__name__)
+
+
+class Py3oPdfOptions(models.Model):
+    _name = 'py3o.pdf.options'
+    _description = 'Define PDF export options for Libreoffice'
+
+    name = fields.Char(required=True)
+    # GENERAL TAB
+    # UseLosslessCompression (bool)
+    image_compression = fields.Selection([
+        ('lossless', 'Lossless Compression'),
+        ('jpeg', 'JPEG Compression'),
+        ], string='Image Compression', default='jpeg')
+    # Quality (int)
+    image_jpeg_quality = fields.Integer(
+        string='Image JPEG Quality', default=90,
+        help="Enter a percentage between 0 and 100.")
+    # ReduceImageResolution (bool) and MaxImageResolution (int)
+    image_reduce_resolution = fields.Selection([
+        ('none', 'Disable'),
+        ('75', '75 DPI'),
+        ('150', '150 DPI'),
+        ('300', '300 DPI'),
+        ('600', '600 DPI'),
+        ('1200', '1200 DPI'),
+        ], string='Reduce Image Resolution', default='300')
+    watermark = fields.Boolean('Sign With Watermark')
+    # Watermark (string)
+    watermark_text = fields.Char('WaterMark Text')
+    # UseTaggedPDF (bool)
+    tagged_pdf = fields.Boolean('Tagged PDF (add document structure)')
+    # SelectPdfVersion (int)
+    # 0 = PDF 1.4 (default selection).
+    # 1 = PDF/A-1 (ISO 19005-1:2005)
+    pdfa = fields.Boolean(
+        'Archive PDF/A-1a (ISO 19005-1)',
+        help="If you enable this option, you will not be able to "
+        "password-protect the document or apply other security settings.")
+    #  ExportFormFields (bool)
+    pdf_form = fields.Boolean('Create PDF Form', default=True)
+    # FormsType (int)
+    pdf_form_format = fields.Selection([
+        ('0', 'FDF'),
+        ('1', 'PDF'),
+        ('2', 'HTML'),
+        ('3', 'XML'),
+        ], string='Submit Format', default='0')
+    # AllowDuplicateFieldNames (bool)
+    pdf_form_allow_duplicate = fields.Boolean('Allow Duplicate Field Names')
+    # ExportBookmarks (bool)
+    export_bookmarks = fields.Boolean('Export Bookmarks', default=True)
+    # ExportPlaceholders (bool)
+    export_placeholders = fields.Boolean('Export Placeholders', default=True)
+    # ExportNotes (bool)
+    export_comments = fields.Boolean('Export Comments')
+    # ExportHiddenSlides (bool) ??
+    export_hidden_slides = fields.Boolean(
+        'Export Automatically Insered Blank Pages')
```

Comment:
"Automatically export Inserted Blank Pages"?

Reply:
I decided to keep the exact label of the Libreoffice interface, so that users can easily find the correspondance between LO and Odoo PDF export options. As you can see in the screenshot avail here https://bitbucket.org/faide/py3o.renderers.juno/issues/1/add-ability-to-set-pdf-export-options this is the exact LO label.

Reply:
strange but :+1: indeed